ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -277,9 +277,8 @@ SettingsContentBase {
             tokensListModel: tokensStore.extendedFlatTokensModel
             baseWalletAssetsModel: RootStore.assets
             baseWalletCollectiblesModel: {
-                RootStore.setFillterAllAddresses() // FIXME no other way to get _all_ collectibles?
                 // TODO concat proxy model to include community collectibles (#12519)
```

Comment:
`TODO` can be removed right?

Reply:
guess so! the model doesn't discriminate against community collectibles